packages/overlays/docs/GlobalOverlayController.md

```diff
@@ -0,0 +1,31 @@
+# GlobalOverlayController
+
+This is a base class for different global overlays (e.g. a dialog, see [Overlay Occurrences](./OverlayOccurrences.md) - the ones positioned relatively to the viewport.
```

Comment:
Above was meant to be posted earlier, but was still in a pending review. Most of it holds still. I will adjust this one and the overlay design documents will be published via rich select branch, which holds the latest version

Reply:
Since the feat/richSelect branch is not being used for development of the rich select anymore, I'm afraid those docs will get lost, so I readded them in this mr :)